apps/web/src/pages/PoolCandidates/ViewPoolCandidatePage/ViewPoolCandidatePage.tsx

```diff
@@ -583,8 +584,21 @@ export const ViewPoolCandidate = ({
       </p>
       <Separator
         data-h2-background-color="base(black.lightest)"
-        data-h2-margin="base(x1, 0, 0, 0)"
+        data-h2-margin="base(x1, 0, x1, 0)"
       />
+      {parsedSnapshot && (
+        <div
+          data-h2-container="base(center, large, 0)"
+          data-h2-text-align="base(right)"
+        >
```

Comment:
Not sure how to fit the button right against the edge like other stuff, I zeroed out padding and negative padding didn't do anything.

![image](https://github.com/GCTC-NTGC/gc-digital-talent/assets/40485260/4ac7da44-be65-48e9-8eed-9c8b1b4f3d1d)

![image](https://github.com/GCTC-NTGC/gc-digital-talent/assets/40485260/855d2e36-3c0a-4ec5-adf9-32ec250f82da)

Reply:
Never mind, it was simple
https://github.com/GCTC-NTGC/gc-digital-talent/pull/8185/commits/e52cb9e9ad830b0e567b82ef41d1639c349acc14